bike-erp/backend/src/test/services/authenticationService/authenticationService.test.ts

```diff
@@ -46,8 +46,8 @@ describe("Login test", () => {
     //Trying to login with the wrong password
     try {
       await AuthenticationService.login("test@test.com", "tes");
```

Comment:
the second string is "tes", was it meant to be "test"?

Reply:
It's just a password. The user password is "test" but I entered "tes" to test a wrong password.

Reply:
Not a problem at all! Thank you for clarifying!